### DISCUSSION TITLE
Modifications to the IFOV?

### DISCUSSION THREAD

**Discussion by mchopping** [STELLA-Landsat/STELLA#29] [Instrument Engineering]
H everyone, My understanding is that STELLA has a 40° IFOV, so the target sampled has a diameter about the same as the altitude above the target. Are there any ways to modify this so that it is much narrower? What are the implications of restricting the IFOV by the addition of a physical sighting mechanism? Thanks in advance for any insights (pun intended).

**Comment by paulmirel** [comment]
For the thermal infrared sensor, there is a 5º FOV option, MLX90614ESF-BCI-000-SP, which is a direct, pin to pin replacement, with no changes needed to the software. 
https://www.digikey.com/en/products/detail/melexis-technologies-nv/MLX90614ESF-BCI-000-SP/5414796

For the spectral sensors, they have an aperture bore that sets the field of view, and as far as I know, there aren't similarly capable sensors at similar price points with smaller fields of view. A plain bore view restrictor will lower the area that the sensor sees, which will lower the optical flux, so, 1. the signal will be smaller, and 2. the calibrated output numbers will be incorrect. You can use the uncalibrated values, the raw analog-to-digital counts, by altering the STELLA operating software. To try to gather more light to keep the signal values large enough to be useful, you can use a lens that is highly transmittive in the wavelengths of that sensor. You'll need an optics tube, and aligning to the sensor will be somewhat challenging, because the sensor device package is small, and square, and the aperture is off-center. Again, the calibrated values will be incorrect, so you'd want to go with the raw counts. So, optical improvements are possible, but not inexpensive. 

We're interested in these adaptations because we would like to more closely match the field of view of the Landsat sensors, and possibly put a version of the STELLA instrument into low Earth orbit on a cubesat, or maybe even a swarm of cubesat, to get unprecedented coverage by time and location for a not-prohibitive cost. 

As a cubesat demonstration, having a ~200km spot size is still workable. A group of students who launch a STELLA-like Earth-looking cubesat will learn to read the spectral signal coming from large areas in a very similar way to learning to read the spectral signals from smaller areas, from more sophisticated sensors. Ocean, desert, forest, ice, city, all will show up in that data stream.

**Reply by mchopping** [reply]
Hi Paul, Thank you. Calibration is not an issue as we can use spectralon reference panels to obtain reflectance factors - but the S/N might well be limiting. I haven't done the precise calculation for the reduction in light reaching the detector by reducing the IFOV (e.g.,) from 40° to 5°, but assuming a completely flat response with angle, it's likely a very small fraction. My knowledge of optics is insufficient for developing the lens solution; hopefully, others will be able to tackle that at some point.

**Comment by mchopping** [comment]
Just a quick query relevant to IFOV size: has anyone characterized the point spread function (PSF) of the STELLA sensors?

**Reply by paulmirel** [reply]
The sensors are single pixel devices, so they don't have a point spread function per se. They average everything in the field of view, which is circular when on-axis. The spot will be an ellipse if the surface in view is at an angle to the line of sight. I haven't seen a spot map, but the aperture is circular and the concentrating lens is circular, so I expect the map to be quite circular. Does this address your thoughts here?

**Reply by mchopping** [reply]
Thanks. I wondered whether the contribution to total measured radiance was less at the edges of the FOV. Tthe answer is no (so my students will have to make absolutely sure their shoes aren't observed along with the target!).

**Comment by paulmirel** [comment]
Here's the document I based my assertions on. https://ams.com/documents/20143/36005/AS726x_AN000490_1-00.pdf/06b6a5bb-6b21-1259-788f-92c7e1405ed6

<img width="894" alt="as726xoptics" src="https://github.com/STELLA-Landsat/STELLA/assets/6273634/07f2e23e-1e3e-417a-8851-5e19ea1b4794">
